get_user_input.py

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/python
+import argparse
+
+def check_pluck_position(string_length, pluck_position):
+
+    if string_length <= pluck_position:
+        print("The position of the pluck is greater than the string length.\nProgram ended.")
+    elif  pluck_position <= 0:
+        print("The position of the pluck is less than zero.\nProgram terminated.")
+        exit()
+    pass
```

Comment:
You don't need to/shouldn't keep `pass` at the end of these functions.  We put that in the stub functions so that they will have correct syntax, but the `pass` command basically means do nothing